SPECS/procps-ng/10-console-messages.conf

```diff
@@ -0,0 +1,2 @@
+# Suppress debug messages on console
+kernel.printk = 4 4 1 7
```

Comment:
I notice that on fedora, they don't provide any overriding sysctl like this, yet i don't see lots of debug printed on their console...have we made sure there isn't some other change we should make?

Reply:
Good point. I thought Fedora doesn't need this because their default kernel printk from the kernel might be set lower than `=7` but this is a guess. Will double-check

Reply:
Confirmed that on f39 workstation, they use their kernel built-in defaults rather than an overriding sysctl. Instead, on the workstation image, by default it boots with `quiet` on the kernel command line, so their kernel's `CONFIG_CONSOLE_LOGLEVEL_QUIET=3` setting takes effect, rather than `CONFIG_CONSOLE_LOGLEVEL_DEFAULT=7`.